### PR TITLE
Create chessboard

### PR DESCRIPTION
**Pull request by aae1990**
Hi, I created the chessboard using javascript and connected the games controller with the view. Additionally the games page is also displaying the game id. Let me know what you think.

### CODE REVIEW

**Comment by Mikeysax** [comment]
@aae1990 You need to run rubocop before you commit/push to your branch. If you run rubocop and you get errors then run rubocop -a you will see how much it autocorrects. What ever it doesn't fix is on you to look up and see why.

**Comment by lasley** [comment]
:heart: static code analyzers (rubocop)

**Comment by aae1990** [comment]
Fixed everything, it's ready to be merged now.

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,102 @@
+
+<h2>Welcome to Game  <<%=@game.id%>></h2>
+<br/>
+
+<table>
+    <div id ="x8">
```
Comment (lasley) [inline app/views/games/show.html.erb]: Seems like there's a lot of repeating going on here. Maybe a few loops to generate this table instead?

Comment (aae1990) [inline app/views/games/show.html.erb]: Ok, I didn't it because of the black/white colours of the cells but will have a look into that.

**Comment by lasley** [comment]
:+1:

**Pull request merged**